Makefile

```diff
@@ -129,7 +129,12 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 .PHONY: local-deploy-with-watcher
 local-deploy-with-watcher: generate kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
-	$(KUSTOMIZE) build config/watcher_local_test | kubectl apply -f -
+	$(KUSTOMIZE) build config/watcher_local_test_with_certmanager | kubectl apply -f -
+
+.PHONY: local-deploy-with-watcher-gcm
+local-deploy-with-watcher-gcm: generate kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Comment:
I think maybe it's time we add some description to replace current comments (`## Deploy controller to the K8s cluster specified in ~/.kube/config.`) for those two commands, it's hard to tell from the command what's the difference and the purpose.